compio-net/src/resolve/windows.rs

```diff
@@ -97,6 +97,7 @@ impl AsyncResolver {
         );
         match res {
             0 => Poll::Ready(Ok(())),
+            x if x == ERROR_IO_PENDING as i32 => Poll::Pending,
```

Comment:
Interesting. `WSA_IO_PENDING` is not a documented return value, and anyway `GetLastError` should work. What is the value of `GetLastError` in this case?